packages/app/src/cli/utilities/extensions/cli.ts

```diff
@@ -68,3 +84,56 @@ export async function nodeExtensionsCLIPath(): Promise<string> {
     return executablePath
   }
 }
+
+/**
+ * This method provides a Writable wraper which parses the incoming chunks as GoLog from Json and writes them back to
+ * the Writable given as parameter. If Parsing to Json fails it wirtes the chunk as it is and thows an error.
+ * @param output {Writable} A Writable which will be wrapped and in which the parsed chunks will be written.
+ * @returns {Writable} A Writable wrapping the parameter.
+ */
+export function goLogWritable(output: Writable): Writable {
+  return new Writable({
+    write(chunk, _encoding, next) {
+      const lines = outputKit.stripAnsiEraseCursorEscapeCharacters(chunk.toString('ascii')).split(/###LOG_END###/)
```

Comment:
have you tried with different logs? do you get chunks that include only part of a log or more than 1 json?
Wondering if the LOG_END is necessary here

Reply:
Nevermind, 🎩 'ed and added some logs and saw that yes you can receive multiple jsons in a chunk